modules/release-notes/ref-release-notes-breaking-changes.adoc

```diff
@@ -131,3 +74,21 @@ Two plugins previously under the @janus-idp scope have moved to @red-hat-develop
 | `@red-hat-developer-hub/backstage-plugin-bulk-import-backend`
 |===
 
+With the update to the plugin scope, the dynamic plugin configuration has also been modified.
+
+[cols=2,%header]
+|===
+|*RHDH 1.3 Configuration*|*RHDH 1.4 Configuration*
+|link:https://github.com/janus-idp/backstage-showcase/blob/release-1.3/dynamic-plugins.default.yaml[dynamic-plugins.default.yaml]|link:https://github.com/janus-idp/backstage-showcase/blob/release-1.4/dynamic-plugins.default.yaml[dynamic-plugins.default.yaml]
+|===
+
+.Procedure
+* To upgrade from {product-very-short} 1.3 to {product-very-short} 1.4, you must update your configuration to use the latest versions of the plugins listed previously from the new scope.
+
+[NOTE]
+====
+Along with the tables provided previously, you can compare the following files to see which dynamic plugins have changed between {product-very-short} 1.3 and {product-very-short} 1.4:
```

Comment:
The added URLs don't seem very user-friendly. Would it be okay to revise the sentence as follows, adding text to the links and removing the bullet points?
```suggestion
In addition to the previously provided tables, you can compare the link:https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/blob/release-1.4/modules/dynamic-plugins/rhdh-supported-plugins.csv[RHDH 1.4 CSV file] with the link:https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/blob/release-1.3/modules/dynamic-plugins/rhdh-supported-plugins.csv[RHDH 1.3 CSV file] to identify the changes in dynamic plugins.
```